components/mpas-ocean/src/shared/mpas_ocn_eddy_parameterization_helpers.F

```diff
@@ -284,11 +284,11 @@ subroutine ocn_eddy_compute_mixed_layer_depth(statePool, forcingPool)
         !$omp end do
         !$omp end parallel
 
-        if (associated(landIceMask) ) then
+        if (associated(landIceFloatingMask) ) then
           !$omp parallel
           !$omp do schedule(runtime)
           do iCell = 1, nCells
-             dThreshMLD(iCell) = dThreshMLD(iCell) - abs(landIceDraft(iCell))*landIceMask(iCell)
+             dThreshMLD(iCell) = dThreshMLD(iCell) - abs(landIceDraft(iCell))*landIceFloatingMask(iCell)
```

Comment:
This is a more troubling case than the analysis members because it is in the eddy parameterization, so it affects the prognostic variables.  Still, I think we want to take out `landIce(Floating)Mask` here as well unless we want to do that in a separate, non-BFB PR.

Reply:
We could have a PR before or after this one that removes these multiplications.

Reply:
The commit is here for now 6f2187fe69. I'll plan on including it unless it makes testing too much of a nightmare.